### ISSUE TITLE
"Abfahrtstabelle" for stationboard

### ISSUE THREAD

**Issue by engenheirolapis** [OpendataCH/Transport#143]
It would be great if the trains etc. arriving at a station could be retrieved in the same manner as the stationboard. The issue from 02.04.12 statet that it is possible the get that by just leaving the from empty, but this doesn't seem to work now..

**Comment by fabian** [comment]
Are you referring to https://github.com/OpendataCH/Transport/issues/35?

**Comment by engenheirolapis** [comment]
yes, exactly #35 [https://github.com/OpendataCH/Transport/issues/35](https://github.com/OpendataCH/Transport/issues/35)

**Comment by fabian** [comment]
Okay, I'm still not 100% sure what you are looking for - could you help me with the following clarifications so I can better understand your request?

In the title of this issue you say "Abfahrtstabelle" (departure) but in the description you are referring to "arrival". Departures for a station can be fetched with `/stationboard`. So I assume you are looking for arrivals at a specific station (as written in the description).

I suspect this is currently not possible because `$boardType = 'DEP'` is hardcoded in [StationBoardQuery](https://github.com/OpendataCH/Transport/blob/4c824fc0cc1f328fb475db9b6df91e9f2d1e7a42/lib/Transport/Entity/Schedule/StationBoardQuery.php#L16). I would think this could be added with an additional parameter - are you interested in sending a pull request for this addition?

Issue #35 is about moving certain functionality over to `/connections` - something you are not necessarily interested in if you can get the information from the `/stationboard`, correct?

**Comment by engenheirolapis** [comment]
Yes, you got it right, I got the title wrong. I like to have a arrivals table exactly like the stationboard but with arrivals.
Looks like its not a big thing to add this functionality. I'll look into it. Seems that it can just be changed to ARR to get the arrivals.
